CMakeLists.txt

```diff
@@ -159,7 +159,7 @@ install(FILES package.xml DESTINATION share/libnabo)
 export(PACKAGE libnabo)
 
 # Create variable with the library location
-get_property(libnabo_library TARGET ${LIB_NAME} PROPERTY LOCATION)
+#get_property(libnabo_library TARGET ${LIB_NAME} PROPERTY LOCATION)
```

Comment:
Are we sure this change and the following are not breaking the the `find_package(libnabo)` feature? It seems to me, who looks to this CMake code for the first time, that the current indirection aims at setting libnabo_library to its absolute path. Is the new version still doing so? I understand that in the current code, the use of a hardcoded `/lib/` might be an error, for instance for dynamic libraries on Windows.